sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -52,6 +52,7 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
 
             var itemCardViewOneCol = new Sourcemash.Views.ItemCardView({el: "#item-" + item.get('id') + "-onecol", model: item, user: user });
             itemCards.push(itemCardViewOneCol);
+            $(".card-reveal a").attr("target","_blank");
```

Comment:
Why does it have to be a script tag? I may be missing something, but can't you just do
``` HTML
<p class="flow-text"><a href="<%= item.get('link') %>" target="_blank"><%= item.get('title') %></a></p>
```